cmd/node/node.go

```diff
@@ -190,9 +190,9 @@ func loadConfig(cmd *cobra.Command) (*config.Config, error) {
 // LoadConfigFromFile tries to load configuration file if the config parameter was specified.
 func LoadConfigFromFile() (*config.Config, error) {
 	fileLocation := viper.GetString("config")
-	vip := viper.New()
```

Comment:
curious why the change to use global viper?

Reply:
because using both requires checking variables in both, global is used when flags are parsed and local for reading config. i don't understand what was the motivation in the first place.